cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Replica.java

```diff
@@ -123,51 +124,58 @@ void clearLoad() {
    * (2) Clear and get the outbound network load associated with leadership from the given replica.
    * (3) Clear and get the CPU leadership load associated with leadership from the given replica.
    *
-   * @return Removed leadership load by snapshot time -- i.e. outbound network and fraction of CPU load by snapshot time.
+   * @return Removed leadership load by windows -- i.e. outbound network and fraction of CPU load by windows.
    */
-  Map<Resource, double[]> makeFollower() {
+  AggregatedMetricValues makeFollower() {
     // Remove leadership from the replica.
     setLeadership(false);
-    // Clear and get the outbound network load associated with leadership from the given replica.
-    double[] leadershipNwOutLoad = _load.loadFor(Resource.NW_OUT).doubleArray();
-    MetricValues leadershipCpuLoad = _load.loadFor(Resource.CPU);
 
+    // Get the inbound/outbound network and cpu load associated with leadership from the given replica.
+    // All the following metric values are in a shared mode to avoid data copy.
+    int cpuMetricId = KafkaMetricDef.resourceToMetricIds(Resource.CPU).get(0);
```

Comment:
Would it be helpful to add a comment to explain why we `get(0)`? -- maybe add a `todo` for future?

Reply:
Replica is a pretty specific Kafka concept, so we do not have todo for the future here.